vulyk/models/__init__.py

```diff
@@ -1,16 +1,15 @@
-# coding=utf-8
-from mongoengine.base.fields import ObjectIdField
+# -*- coding=utf-8 -*-
+
 from bson import ObjectId
+from mongoengine.base.fields import ObjectIdField
 
-from vulyk.models.user import User, Group
-from vulyk.models.tasks import AbstractTask, AbstractAnswer
-from vulyk.models.stats import WorkSession
+import six
```

Comment:
six should go higher in the order I think?
(base libs, 3rd part libs, framework libs, in-house stuff)